### PR TITLE
fix(DB/Quest): Text for quest "When the Cows Come Home"

### PR DESCRIPTION
**Pull request by Hacaw**
## CHANGES PROPOSED:
- Added missing texts in database for Bessy
- Added missing script lines and messages for the Thadell

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3770

## TESTS PERFORMED:
- All database scripts run without error
- No build errors
- Tested on Windows with Docker

## HOW TO TEST THE CHANGES:
- finish first quest from Thadell (find the Bessy first)
- .go creature Bessy
- take quest, escort and protect the cow 
- observe that Bessy is no longer talking like a human and it only moos (duh...)
- when Bessy arrives at Thadell, Thadell will thank you with 2 messages

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+INSERT INTO `broadcast_text` (`ID`, `Language`, `MaleText`, `FemaleText`, `EmoteID0`, `EmoteID1`, `EmoteID2`, `EmoteDelay0`, `EmoteDelay1`, `EmoteDelay2`, `SoundId`, `Unk1`, `Unk2`, `VerifiedBuild`)VALUES 
+(100003, 0, 'Mooooo...', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
+
+DELETE FROM `broadcast_text` WHERE `id` = 100004;
+INSERT INTO `broadcast_text` (`ID`, `Language`, `MaleText`, `FemaleText`, `EmoteID0`, `EmoteID1`, `EmoteID2`, `EmoteDelay0`, `EmoteDelay1`, `EmoteDelay2`, `SoundId`, `Unk1`, `Unk2`, `VerifiedBuild`)VALUES 
+(100004, 0, 'Moooooooooo!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
+
+DELETE FROM `broadcast_text` WHERE `id` = 100006;
+INSERT INTO `broadcast_text` (`ID`, `Language`, `MaleText`, `FemaleText`, `EmoteID0`, `EmoteID1`, `EmoteID2`, `EmoteDelay0`, `EmoteDelay1`, `EmoteDelay2`, `SoundId`, `Unk1`, `Unk2`, `VerifiedBuild`)VALUES 
+(100005, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1606921345291246500.sql:14]: You can combine all the changes to `broadcast_text` together:

```sql
DELETE FROM `broadcast_text` WHERE `id` IN (100003, 100004, 100006);
INSERT INTO `broadcast_text` (`ID`, `Language`, `MaleText`, `FemaleText`, `EmoteID0`, `EmoteID1`, `EmoteID2`, `EmoteDelay0`, `EmoteDelay1`, `EmoteDelay2`, `SoundId`, `Unk1`, `Unk2`, `VerifiedBuild`)VALUES 
(100003, 0, 'Mooooo...', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019),
(100004, 0, 'Moooooooooo!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019),
(100005, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
```

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+DESC creature_text;
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1606921345291246500.sql:17]: ?

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+(20415, 0, 0, 'Mooooo...', 12, 0, 100, 0, 0, 0, 100003, 0, 'Bessy calmly moo when quest starts');
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(20415, 1, 0, 'Moooooooooo!', 12, 0, 100, 0, 0, 0, 100004, 0, 'Bessy angry moo when being attacked');
+
+-- update Thadell messages
+DELETE FROM `creature_text` WHERE `creatureId` = 20464;
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(20464, 2, 0, 'Bessy, you''re home. Thank the Light!', 12, 0, 100, 0, 0, 0, 18181, 0, 'Thadell say to Bessy');
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(20464, 3, 0, 'Thank you for bringing back my Bessy, $N. I couldn''t live without her!', 12, 0, 100, 0, 0, 0, 100005, 0, 'Thadell say to player');
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1606921345291246500.sql:30]: same as above, they can be combined in one `DELETE`/`INSERT` statement

**Comment by FrancescoBorzi** [comment]
@Hacaw thanks for this contribution! Just some minor codestyle remarks

**Comment by Si1ker** [comment]
@Hacaw  Use this method for reference, ju only need 1 delete, 1 insert and then just add what you're gonna add, don't need to do an insert for each one.
https://github.com/azerothcore/azerothcore-wotlk/pull/3797/files#diff-137df1ce3898873784b9c2e6058ad55d3418bb907ea5cc435b10d8749efc227dR57

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql (hunk 1 above)
Comment (Hacaw) [reply data/sql/updates/pending_db_world/rev_1606921345291246500.sql:8]: Fixed, you're right. Thanks 👍

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql (hunk 2 above)
Comment (Hacaw) [reply data/sql/updates/pending_db_world/rev_1606921345291246500.sql]: Sorry, leftover from debugging. Removed now.

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql (hunk 3 above)
Comment (Hacaw) [reply data/sql/updates/pending_db_world/rev_1606921345291246500.sql:16]: Fixed.

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by iyasenai** [comment]
I tested it, watch the video
https://www.youtube.com/watch?v=RxGR4-Nw4kw

**Comment by Hacaw** [comment]
> I tested it, watch the video
> https://www.youtube.com/watch?v=RxGR4-Nw4kw

Just watched the video, nice job and thanks for testing it !

The only issue would be the second message from Thadell should have been in Italian based on your client, I guess I should have updated the broadcast_text_locale table for Thadell entry, but don't know the correct phrasing in Italian :D.

**Comment by Si1ker** [comment]
@iyasenai Thanks for testing !
@Hacaw His client is Spanish, not Italian.

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+(100004, 0, 'Moooooooooo!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019),
+(100005, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
+
+-- update Bessy & Thadell messages
+DELETE FROM `creature_text` WHERE `creatureId` IN (20415, 20464);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(20415, 0, 0, 'Mooooo...', 12, 0, 100, 0, 0, 0, 100003, 0, 'Bessy calmly moo when quest starts'),
+(20415, 1, 0, 'Moooooooooo!', 12, 0, 100, 0, 0, 0, 100004, 0, 'Bessy angry moo when being attacked'),
+(20464, 2, 0, 'Bessy, you''re home. Thank the Light!', 12, 0, 100, 0, 0, 0, 18181, 0, 'Thadell say to Bessy'),
+(20464, 3, 0, 'Thank you for bringing back my Bessy, $N. I couldn''t live without her!', 12, 0, 100, 0, 0, 0, 100005, 0, 'Thadell say to player');
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1606921345291246500.sql:16]:
```suggestion
(20464, 3, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', 12, 0, 100, 0, 0, 0, 100005, 0, 'Thadell say to player');

```

**Review by Si1ker — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1606921345291246500.sql [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+(100004, 0, 'Moooooooooo!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019),
+(100005, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', '', 5, 0, 0, 0, 0, 0, 0, 0, 1, 18019);
+
+-- update Bessy & Thadell messages
+DELETE FROM `creature_text` WHERE `creatureId` IN (20415, 20464);
+INSERT INTO `creature_text` (`CreatureID`, `GroupID`, `id`, `text`, `type`, `language`, `Probability`, `Emote`, `Duration`, `Sound`, `BroadcastTextId`, `TextRange`, `comment`) VALUES
+(20415, 0, 0, 'Mooooo...', 12, 0, 100, 0, 0, 0, 100003, 0, 'Bessy calmly moo when quest starts'),
+(20415, 1, 0, 'Moooooooooo!', 12, 0, 100, 0, 0, 0, 100004, 0, 'Bessy angry moo when being attacked'),
+(20464, 2, 0, 'Bessy, you''re home. Thank the Light!', 12, 0, 100, 0, 0, 0, 18181, 0, 'Thadell say to Bessy'),
+(20464, 3, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', 12, 0, 100, 0, 0, 0, 100005, 0, 'Thadell say to player');
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1606921345291246500.sql:16]:
```suggestion
(20464, 3, 0, 'Thank you for bringing back my Bessy, $N. I couldn\'t live without her!', 12, 0, 100, 0, 0, 0, 100005, 0, 'Thadell say to player');


```

**Review by Si1ker — COMMENTED** [review]
re-do new line at the end, github didn't take it from the last change.

**Review by Si1ker — APPROVED** [review]

**Comment by Hacaw** [comment]
When is going to be merged ?

**Comment by FrancescoBorzi** [comment]
> When is going to be merged ?

https://www.azerothcore.org/wiki/merge-process

**Pull request merged**